include/nuttx/efuse/esp_efuse_table.h

```diff
@@ -0,0 +1,69 @@
+/****************************************************************************
+ * include/nuttx/efuse/esp_efuse_table.h
```

Comment:
should we put the header file to arch/xtensa/include/esp32?

Reply:
I don't think so, this header file is exported to applications and this is just an export mirror of the arch/xtensa/src/esp32/esp32_efuse_table.c

Reply:
Header files put in nuttx/include can be called from any chip which different arch, but esp32_efuse_table.h can only be used on esp32. That's why is better to put into arch/xtensa/esp32/include. BTW, it is no problem to include the public arch header files from application, I think.

Reply:
Done!